eo-maven-plugin/src/main/java/org/eolang/maven/BinarizeMojo.java

```diff
@@ -80,17 +80,17 @@ public final class BinarizeMojo extends SafeMojo {
     private File generatedDir;
 
     /**
-     * The directory with eo_env rust project. It is a necessary dependency
+     * The directory with portal project. It is a necessary dependency
      * that provides rust-eo interaction.
      * @checkstyle MemberNameCheck (8 lines)
      */
     @Parameter(
-        property = "eo.env",
+        property = "eo.portal",
```

Comment:
@levBagryansky Please add a description of what a `Portal` is to avoid the ambiguity of this concept

Reply:
@Graur The description already exists. See above

>  It is a necessary dependency
>      * that provides rust-eo interaction.